BruteShark/PcapAnalyzer/PcapAnalyzer.csproj

```diff
@@ -1,12 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Asn1DotNet" Version="1.0.2" />
+    <PackageReference Include="Asn1DotNet" Version="1.0.3" />
+    <PackageReference Include="System.Text.Json" Version="5.0.0" />
```

Comment:
I think System.Text.Json is part of .Net standard library and there is no need to explicit reference it (I'm not sure)

Reply:
system.text. json is a part of .net core but must be installed manually in .net standard